app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -754,6 +762,7 @@ class Product extends \Magento\ImportExport\Model\Import\Entity\AbstractEntity
      * @param array $data
      * @param array $dateAttrCodes
      * @param CatalogConfig $catalogConfig
+     * @param StockItemImporterInterface $stockItemImporter
```

Comment:
Order of parameters in the docblock is different from the method signature

Reply:
done.